components/provisioner/internal/runtime/configuration_test.go

```diff
@@ -29,7 +29,7 @@ func TestProvider_CreateConfigMapForRuntime(t *testing.T) {
 	tenant := "tenant"
 	token := "shdfv7123ygfbw832b"
 
-	namespace := "compass-system"
+	namespace := "kcp-system"
```

Comment:
```suggestion
	namespace := "compass-system"
```
Compass Runtime Agent is still installed in `compass-system` so maybe let's leave it as it was to avoid confusion.